### PR TITLE
EUI-8727 Register other org - New terms and conditions

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8727

### Change description ###
Register other org - New Terms and Conditions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]

File: src/app/app.routes.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { AuthGuard } from '../user-profile/guards/auth.guard';
 import { AccessibilityComponent, CookiePolicyComponent, GetHelpComponent, PrivacyPolicyComponent, ServiceDownComponent, SignedOutComponent, TermsAndConditionsComponent } from './components';
 import { RedirectComponent } from './containers/redirect/redirect.component';
 import { TermsConditionGuard } from './guards/termsCondition.guard';
+import { TermsAndConditionsRegisterOtherOrgComponent } from './components/terms-and-conditions-register-other-org/terms-and-conditions-register-other-org.component';
```
Comment (connorpgpmcelroy) [inline src/app/app.routes.ts:10]: incorrect import ordering

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -84,6 +85,10 @@ export const ROUTES: Routes = [
     path: 'terms-and-conditions',
     component: TermsAndConditionsComponent
   },
+  {
+    path: 'terms-and-conditions-register-other-org',
+    component: TermsAndConditionsRegisterOtherOrgComponent
```
Comment (connorpgpmcelroy) [inline src/app/app.routes.ts:90]: Minor comment: Really long component name and path. Could we shorten it?

File: src/app/components/index.ts [hunk 3]
```diff
@@ -16,17 +17,19 @@ export const components: any[] = [
...
   ServiceDownComponent,
   GetHelpComponent,
+  TermsAndConditionsRegisterOtherOrgComponent,
   TermsAndConditionsComponent,
   SignedOutComponent
 ];
 
-export * from './hmcts-global-header/hmcts-global-header.component';
-export * from './hmcts-global-footer/hmcts-global-footer.component';
 export * from '../../../projects/gov-ui/src/lib/components/hmcts-primary-navigation/hmcts-primary-navigation.component';
```
Comment (connorpgpmcelroy) [inline src/app/components/index.ts:25]: not your code but src mentioned here

File: src/app/components/terms-and-conditions/terms-and-conditions.component.spec.ts [hunk 4]
```diff
@@ -7,6 +7,8 @@ import { Store } from '@ngrx/store';
 import { of } from 'rxjs';
 import { TermsConditionsService } from '../../../shared/services/termsConditions.service';
 import { TermsAndConditionsComponent } from './terms-and-conditions.component';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { Router } from '@angular/router';
```
Comment (connorpgpmcelroy) [inline src/app/components/terms-and-conditions/terms-and-conditions.component.spec.ts:11]: incorrect import ordering

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor changes requested (not necessary to change component name but just thought I'd raise it as unusual)

File: src/app/app.routes.ts (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply src/app/app.routes.ts:90]: I started the development with TermsAndConditions**New**Component, but it did not sound like a suitable name so changed it to this lengthy name.

We will leave it to the future developers to rename when the Business wants to get rid of the current Terms and Conditions and use this new one instead :)

File: src/app/components/index.ts (hunk 3 above)
Comment (johnbenjamin-hmcts) [reply src/app/components/index.ts:25]: The `projects > gov-ui` is a separate project that lives at the same level as `src`.

**Review by Daniel-Lam — APPROVED** [review]
@johnbenjamin-hmcts - all looks OK here; approved

File: src/app/app.routes.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/app/app.routes.ts]: Thank you @connorpgpmcelroy I updated the code now.

File: src/app/components/terms-and-conditions/terms-and-conditions.component.spec.ts (hunk 4 above)
Comment (johnbenjamin-hmcts) [reply src/app/components/terms-and-conditions/terms-and-conditions.component.spec.ts]: Thank you @connorpgpmcelroy I updated the code now.

File: src/app/components/index.ts (hunk 3 above)
Comment (connorpgpmcelroy) [reply src/app/components/index.ts:25]: Oh ok, necessary then I suppose

**Review by connorpgpmcelroy — DISMISSED** [review]
All looks good now

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Pull request merged**